src/aws_metadata.app.src

```diff
@@ -5,7 +5,8 @@
   {mod, {aws_metadata_app, []}},
   {applications,
    [kernel,
-    stdlib
+    stdlib,
+    aws
```

Comment:
Ah, right, thanks for explaining.  I always forget about how relx
pulls dependencies in Erlang.